meta-balena-common/recipes-support/hostapp-update-hooks/files/0-signed-update

```diff
@@ -40,6 +41,136 @@ umountEfiVars() {
 }
 
 updateKeys() {
+    PCRS="0,2,3,7"
```

Comment:
@alexgg Are you sure? This hook looks to be solely part of the UEFI secure boot implementation, and it looks untouched by the [Raspberry Pi](https://github.com/balena-os/balena-raspberrypi/pull/1095) secure boot branch. 

I only added the re-locking portion for supporting PCR7 sealing, the structure is the same.

Reply:
@jakogut if it's only EFI, then let's include it conditionally on `MACHINE_FEATURES` containing `efi` instead of [unconditionally](https://github.com/balena-os/meta-balena/blob/ae98ddffb1a8055d98b2104db2f2901e9eddc14c/meta-balena-common/recipes-support/hostapp-update-hooks/hostapp-update-hooks.bb#L11).
No need to add the hooks for platforms that won't use it.